### DISCUSSION TITLE
Live streaming Loki logs on dashboard?

### DISCUSSION THREAD

**Discussion by hterik** [grafana/grafana#39849] [Dashboards, Panels & Visualizations]
Hi
Under the **Explore**-tab, it is possible to get live streaming of logs from Loki, using the Live button in the top right corner.

However, if i on a **dashboard** add a **Logs-panel** connected to a Loki datasource, i can't find any way to enable live streaming there. Closest i can get is to set now-1m time range and very short auto-refresh but that's not the same thing. 
Am i missing something or is this not possible today?

(Grafana v8.1.3)

**Comment by apockill** [comment]
This seems pretty critical for using grafana+loki as a logging solution. The Explore tab has several features that I can't seem to replicate in a dashboard. The live logging is one, but it seems like there's also a "show context" feature next to filtered logs, which would be tremendously helpful.

**Comment by ivanahuckova** [comment]
For discussion about this feature, use https://github.com/grafana/grafana/issues/77551
